src/classlibnative/float/Float.nativeproj

```diff
@@ -28,7 +28,7 @@
   </PropertyGroup>
   <!--Leaf Project Items-->
   <ItemGroup>
-    <CppCompile Include="FloatNative.cpp" />
+    <CppCompile Include="FloatDouble.cpp" />
```

Comment:
The casing on the file name differs here from that used on the actual file. Should be fine for Windows, but is this not a problem for Linux type systems?